### PR TITLE
Issue #2744353 by orbmantell: Fixing cache corruption

### PR DESCRIPTION
**Pull request by ian-rose**
https://www.drupal.org/node/2744353

### CODE REVIEW

File: modules/fb_instant_articles_display/fb_instant_articles_display.module [hunk 1]
```diff
@@ -78,31 +73,27 @@ function fb_instant_articles_display_module_implements_alter(&$implementations,
...
-    if (!function_exists("fb_instant_articles_display_preprocess_{$entity_type}")) {
-      $code .= 'function fb_instant_articles_display_preprocess_' . $entity_type . "(&" . '$vars' . ") {
-      if (isset(" . '$vars' . "['view_mode']) && " . '$vars' . "['view_mode'] === 'fb_instant_article') {
-        " . '$vars' . "['theme_hook_suggestions'][] = 'entity__fb_instant_article';
-      }
-    }";
+function fb_instant_articles_display_preprocess(&$variables, $hook) {
+  $entities = array_keys(entity_get_info());
+  if (isset($variables['elements']['#entity_type']) && in_array($variables['elements']['#entity_type'], $entities, TRUE)) {
+
```
Comment (scottrigby) [inline modules/fb_instant_articles_display/fb_instant_articles_display.module]: Let's kill this extra linebreak.

**Comment by scottrigby** [comment]
@ian-rose Otherwise as I said on d.o this is a nice cleanup!

**Comment by ian-rose** [comment]
Updated.

**Comment by scottrigby** [comment]
@ian-rose Looks good! Except the commit message is wrong – instead of:
`Issue #2734403 by logmill, scottrigby: rss feed fix node operation (#68)`
It should be something like:
`Issue #2744353 by orbmantell, scottrigby: entity_get_info call inside hook_module_implements_alter causes cache corruption (#70)`
Otherwise there will be incorrect references in d.o & gh.

**Comment by ian-rose** [comment]
Rebase fail. Fixed.

**Pull request merged**